journalpump/journalpump.py

```diff
@@ -650,6 +709,10 @@ def configure_field_filters(self):
         filters = self.config.get("field_filters", {})
         self.field_filters = {name: FieldFilter(name, config) for name, config in filters.items()}
 
+    def configure_unit_log_levels(self):
+        unit_log_levels = self.config.get("unit_log_levels", {})
```

Comment:
```suggestion
        unit_log_levels = self.config.get("unit_log_levels", [])
```

Reply:
see comment above. I think this is how we want it to be